src/libraries/System.Private.Uri/src/System.Private.Uri.csproj

```diff
@@ -3,7 +3,7 @@
     <AssemblyName>System.Private.Uri</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Would it make sense to treat the Unix implementation as `$(NetCoreAppCurrent)` instead of `$(NetCoreAppCurrent)-Unix` that way you kill two birds with one stone (`browser` and simplifying P2Ps)

Reply:
This definitely requires a comment. I would still want to see the discussed hook in the TargetFramework.Sdk as I need it in my PR as well (shim -> System.Data.Odbc).